src/Shared/Helpers/IpfsAddressValidator.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace StrixMusic.Helpers
+{
+    /// <summary>
+    /// Validates ipfs addresses.
+    /// </summary>
+    public class IpfsAddressValidator
```

Comment:
This isn't needed. `Ipns` addresses can also be treated as `Cid`s. the `Cid` type validates whatever you give it on construction. 

The code to check CIDs also limits you to CIDv0. We need to be supported CIDv1 and all possible multihashes (included libp2p public key hashes, the ones used for ipns).

Let's just use the `Cid` type provided by `Ipfs.Core`.